storage/client/client.go

```diff
@@ -1579,8 +1579,8 @@ func (c *StorageClient) RuntimeEVMNFTs(ctx context.Context, limit *uint64, offse
 		var contractAddrContextVersion int
 		var contractAddrData []byte
 		var tokenType sql.NullInt32
-		var ownerAddrContextIdentifier string
-		var ownerAddrContextVersion int
+		var ownerAddrContextIdentifier *string
```

Comment:
Let's add a short inline comment here explaining why this is nullable.

Reply:
ya

oo come to think of it we should use `if nft.Owner != nil` instead